udfs/CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing UDFs
+
+Thank you for taking the time to contribute to this repository of user-defined functions (UDFs) for BigQuery.
+
+The following is a set of guidelines for contributing a UDF to this repository.
+
+## Pull Request Process
+
+1. In order to contribute a UDF to this repository, please include these changes in a pull request:
+    * Add a SQL file containing the UDF and a Contributor License Agreement (see the below [section](#contributor-license-agreement) for more information) to the appropriate directory. If this is a function replicating something in a data warehouse, choose the associated sub-directory in the [migration](/udfs/migration) directory. Otherwise, choose the [community](/udfs/community) directory.
+    * Edit the `test_cases.yaml` file to include test inputs and expected outputs for the function. Make sure test cases provide full coverage of the function's expected behavior. For example, if integers are expected input, please provide test cases with the following inputs: negative numbers, zero, positive numbers, and null values.
+    * Edit the `README.md` in the associated sub-directory to include a description of the function. Make sure your function is listed in alphabetical order amongst the other functions in the `README.md`. 
+2. Test your UDF locally using the test cases you added to the `test_cases.yaml` file. Please follow the instructions in the [Testing UDFs Locally section](#testing-udfs-locally) to automatically test all inputs for the expected outputs using the function.
+3. Submit a pull request and we will review the code as soon as possible. Please see the section on [Code Reviews](#code-reviews) for more information.
+
+## Testing UDFs Locally
+
+Please follow these instructions to confirm that the test cases being provided in your pull request work as expected.
+
+0. Change into the bigquery_utils top-level directory.
+1. Create a Python virtual environment.
+2. Run this command to install all requirements: `python3 -m pip install -r udfs/tests/requirements.txt`
+3. Temporarily create a separate directory at the same level where your function currently exists. Place the function inside this directory.
+4. Edit the test_DIRECTORY_udf.py file's UDFS_DIR_PATH variable, placing in the path of your temporarily created directory containing the SQL file with your UDF.
+5. Run this command after replacing DIRECTORY with the correct directory (community or the EDW named directory): `#python3 -m nose2 -c udfs/tests/unittest.cfg test_DIRECTORY_udf` 
```

Comment:
I think this test_DIRECTORY_udf file was removed by latest commit. 
I think these docs just need to be updated.

Reply:
Updates to come.